doc/articles/splash-screen.md

```diff
@@ -2,89 +2,179 @@
 
 This article covers how to add a splash screen to your application.
 
-As a first step, navigate to `YourProject.Shared > Assets` and add all [scales](#table-of-scales) of your splash screen image.
+> [!TIP]
+> The complete source code that goes along with this guide is available in the [unoplatform/Uno.Samples](https://github.com/unoplatform/Uno.Samples) GitHub repository - [SplashScreenSample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SplashScreenSample)
 
-## UWP
+## Prerequisites
```

Comment:
I think we should redirect to the common prerequisites documentation, it may get outdated quickly.